transmission_interface/include/transmission_interface/transmission_interface_loader.h

```diff
@@ -155,7 +155,7 @@ struct InverseTransmissionInterfaces
 
 struct TransmissionLoaderData
 {
-  typedef boost::shared_ptr<Transmission> TransmissionPtr;
+  typedef std::shared_ptr<Transmission> TransmissionPtr; // DEPRECATED and unused!
```

Comment:
remove?

Reply:
Great catch, I've actually forgot about it